modules/@apostrophecms/ui/ui/apos/components/AposButton.vue

```diff
@@ -1,41 +1,42 @@
 <template>
-  <component
-    :is="href ? 'a' : 'button'"
-    v-on="href ? {} : {click: click}"
-    :href="href.length ? href : false"
-    class="apos-button"
-    :class="modifierClass"
-    :tabindex="tabindex"
-    :disabled="isDisabled"
-    :type="buttonType"
-    :role="role"
-    v-tooltip="tooltip"
-    :id="attrs.id ? attrs.id : id"
-    v-bind="attrs"
-  >
-    <transition name="fade">
-      <AposSpinner :color="spinnerColor" v-if="busy" />
-    </transition>
-    <span
-      v-if="colorStyle"
-      class="apos-button__color-preview"
+  <span v-tooltip="tooltip">
```

Comment:
Why this?

Reply:
without this we can't get the proper browser mouse events to fire a tooltip on a disabled button